src/chrome/content/rules/Sina.com.cn.xml

```diff
@@ -135,7 +134,7 @@
 	<exclusion pattern="^http://beacon\.sina\.com\.cn/$" /><!-- 404 -->
 		<test url="http://beacon.sina.com.cn/a.gif" />
 	<target host="corp.sina.com.cn" />
-	<!--target host="d0.sina.com.cn" />
+	<target host="d0.sina.com.cn" />
```

Comment:
Targets `d[0-9].sina.com.cn` fail the fetch test with a TLS handshake error, please comment them out.